agr_literature_service/api/schemas/topic_entity_tag_schemas.py

```diff
@@ -0,0 +1,74 @@
+from typing import List, Optional
+from pydantic import BaseModel
+
+
+class TopicEntityTagPropSchemaCreate(BaseModel):
+    topic_entity_tag_id: Optional[int] = None  # created on the fly so not needed here
+    qualifier: str
+    date_created: Optional[str]
+    date_updated: Optional[str]
+    created_by: Optional[str]
+    updated_by: Optional[str]
+
+
+class TopicEntityTagPropSchemaShow(BaseModel):
+    topic_entity_tag_prop_id: int
+    topic_entity_tag_id: int
+    qualifier: str
+    date_created: str
+    date_updated: Optional[str]
+    created_by: str
+    updated_by: Optional[str]
+
+
+class TopicEntityTagPropSchemaUpdate(BaseModel):
+    qualifier: str
+    topic_entity_tag_prop_id: int
+
+
+class TopicEntityTagSchemaCreate(BaseModel):
+    reference_curie: str
+    topic: str
+    entity_type: str
+    alliance_entity: Optional[str] = None
+    mod_entity: Optional[str] = None
+    new_entity: Optional[str] = None
+    taxon: int
```

Comment:
I think taxon should be a string, because we'd be storing the NCBI taxon id, which is a curie.  The only value we'll be using for the current ticket is NCBITaxon:559292

Reply:
AHH! I was expecting just the number bit i.e. 559292. That is what is normal for taxon ids. But I can change it, as it looks like you want to store the curie. Other option is to strip off the 'NCBITaxon:' but I guess if the A-team is returning the string version, then we should store as is. I presume this is due to them storing this in a xref type table.

Reply:
I think that because it's an ontology term it's keying off the full curie.  You're right though, at Caltech we just store the numeric part of the taxon curie in the taxon table.  I'll ask them to make sure.  Also, editing the comment to mention we need mod_taxon routers, and I would prefer a crud so we don't have to write sqlalchemy to add/delete/patch, but also need a get endpoint that takes a mod to return a list of species, and I'm not sure what that should be called.

Reply:
Lets add mod_taxon router stuff as a separate story later. Presently this is not needed and feels a bit like a waste of time for something so small at the moment. I will let the PO set the priority etc...

Reply:
Sounds good